client/components/room/Video.jsx

```diff
@@ -68,6 +68,10 @@ class Video extends React.Component {
     this.connectSockets();
   }
 
+  componentWillUnmount() {
+    // this.stop()
+  }
```

Comment:
Yeah, I think @dshelters was just testing this when we were looking at the code together. As I mentioned during that chat, `this.stop()` will throw an error as it can't seem to access `pc.close()` since `pc` is `undefined` within the function. So should be good to leave this out for now.